frontend/ui-pablo/app/components/Organisms/auction/AuctionPriceChart.tsx

```diff
@@ -144,13 +143,18 @@ export const AuctionPriceChart: React.FC<AuctionPriceChartProps> = ({
 
   useEffect(() => {
     setOptions(chartOptions(color || theme.palette.primary.main, dateFormat));
-  }, []);
+    // Do not add chart options as dependency as it causes rerendering issues.
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [color, theme, dateFormat]);
```

Comment:
Could we investigate what is the cause of rerendering issues, since this way we won't update the effect if `chartOptions` legitimately changes.